tests/middlewares/roleCheck.spec.ts

```diff
@@ -134,7 +134,7 @@ describe("RoleCheck testing", function () {
         );
         validateJWT(request, response, nxtFunc);
 
-        response.locals.course_id = courseId;
+        response.locals.course_id = 2;
```

Comment:
```suggestion
        response.locals.course_id = courseId;
```

Reply:
Pretty sure at denne burde være course 1 da teacheren ikke har en enrollement i course 2 og så skal fejlbeskeden i asserten andres til "You don't have the authorized role for this action".
 - men honestly tror jeg der er et par fejl i testene tbh - fordi vi giver 2 forskellige fejl beskeder men samme fejlcode. Så i alle deny test er det den forkerte fejlbesked vi tjekker ift. Forstår ik hvordan de andre test ikke fejler tho